### PR TITLE
[cob4-2] Sensorring with asus camera

### PR DESCRIPTION
**Pull request by NadiaHG**

### CODE REVIEW

File: cob_hardware_config/cob4-2/config/image_flip_params_sensorring_cam3d.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# double
+rotation_angle: 90
+
+# reference world coordinate frame with z-axis pointing upwards (= against gravity direction)
+# string
+reference_frame: base_link
+
+# camera coordinate frame (image coordinate system with x-axis to the right, y-axis downwards, z-axis into viewing direction of the camera)
+# string
+camera_frame: cam3d_rgb_optical_frame
```
Comment (NadiaHG) [inline cob_hardware_config/cob4-2/config/image_flip_params_sensorring_cam3d.yaml:15]: this parameter looks wrong, but I get an error if I set it as sensorring_cam3d_frame...

**Comment by NadiaHG** [comment]
Merge together with:
https://github.com/ipa320/cob_calibration_data/pull/92
https://github.com/ipa320/cob_common/pull/175

File: cob_hardware_config/cob4-2/config/image_flip_params_sensorring_cam3d.yaml (hunk 1 above)
Comment (fmessmer) [inline cob_hardware_config/cob4-2/config/image_flip_params_sensorring_cam3d.yaml:15]: what is the error? can you copy-paste it?

File: cob_bringup/robots/cob4-2.xml [hunk 2]
```diff
@@ -200,6 +200,25 @@
 		</include-->
 	</group>
 
+	<group>
+		<machine name="cob4-2-s1" address="$(arg cob4-2-s1)" env-loader="$(find cob_bringup)/env.sh" default="true" timeout="30"/>
+		<!-- pc monitor -->
+		<include file="$(find cob_bringup)/tools/pc_monitor.launch" >
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-2.xml:206]: we probably have to update diagnostics configuration to include cob4-2-s1 pc in the PC diagnostics.

Comment (NadiaHG) [inline cob_bringup/robots/cob4-2.xml:206]: Yes, you are right. I updated the diagnostics configuration

**Comment by floweisshardt** [comment]
LGTM

**Comment by fmessmer** [comment]
Still, I would like to know what the error was!
See my [comment](https://github.com/ipa320/cob_robots/pull/349#discussion_r44039215) above....

**Pull request merged**